frontend/app/views/fragments/form/submit.scala.html

```diff
@@ -44,3 +47,11 @@
         </div>
     </div>
 </fieldset>
+
+@countryGroup.map{ cg =>
+    @if(Seq("us","um","vi").contains(cg.id.toLowerCase)) {
```

Comment:
This line doesn't work quite as might have been intended- the `"um"` and `"vi"` terms will never be activated, because we're looking at the id of a `CountryGroup` here, not a `Country` - there are only 6 CountryGroups  _(UK, US, Canada, Australia, NewZealand, Europe, RestOfTheWorld)_, and none of them have `"um"` and `"vi"` as their id.

However, the line _will_ match the `US` country group, so works reasonably well - and when the user selects their country in the form, the javascript will update the visibility of the line appropriately.

On a related note, when [`CountryGroup` was introduced](https://github.com/guardian/membership-common/commit/8d0d5972accd3a78fa166947cefbcb50a02ac3b3#diff-e19ee1b21c6b25cad28d96066988ab86R261), these countries should have been included in the `US` CountryGroup, not _RestOfTheWorld_:

* `Country("VI", "United States Virgin Islands")`
* `Country("VC", "Saint Vincent & The Grenadines")`